src/Service/CloudFormation/src/Result/DescribeStacksOutput.php

```diff
@@ -100,7 +100,7 @@ protected function populateResult(Response $response): void
         $data = new \SimpleXMLElement($response->getContent());
         $data = $data->DescribeStacksResult;
 
-        $this->stacks = !$data->Stacks ? [] : $this->populateResultStacks($data->Stacks);
+        $this->stacks = null === $data->Stacks ? null : ([] === $data->Stacks ? [] : $this->populateResultStacks($data->Stacks));
```

Comment:
This change looks wrong.
- stacks is not nullable
- PHPStan is write, we can't use `=== []` with an XML node

Reply:
Yepp. You are correct. The PR is updated.